src/main/java/Duke.java

```diff
@@ -1,10 +1,269 @@
+                    list.addTaskList(task);
+                }
+            }
+            writeDukeTextFile(list);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    public static void writeDukeTextFile(TaskList list) {
+        File dukeTextFile = new  File("Duke.txt");
+        try {
+            FileWriter dukeTextWriter = new FileWriter(dukeTextFile);
+            int lineNumber = list.getTaskListCount();
+            for (int i = 0 ; i < lineNumber ; i++) {
+                dukeTextWriter.write(list.textString(i));
+            }
+            dukeTextWriter.close();
+        } catch (Exception e) {
+            e.getStackTrace();
+        }
+
+    }
+
+
+
+    public static void printGreeting() {
+        String logo = "      ____        _        \n"
+                + "     |  _ \\ _   _| | _____ \n"
+                + "     | | | | | | | |/ / _ \\\n"
+                + "     | |_| | |_| |   <  __/\n"
+                + "     |____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(LINE_SEPARATOR);
+        System.out.println("    Hello! I'm Duke");
+        printCommandGuide();
+    }
+
+
+    public static void printCommandGuide(){
+        //prints list of commands
+        System.out.println("    Command Guide");
+        System.out.println("    use \"todo\" (with a space) to add a task without any date or time attached to it");
+        System.out.println("    use \"deadline\" (with a space) and \"/by\" (with a space) to add a task to be done before a certain date");
+        System.out.println("    use \"event\" (with a space) and \"/at\" (with a space) to add a task that start and ends at a specific time");
+        System.out.println("    use \"list\" to show task list");
+        System.out.println("    use \"done\" (with a space) followed by task number to mark a task as done");
+        System.out.println("    use \"bye\" to exit the chat bot");
+        System.out.println("    use \"delete\" followed by the taskNumber in the last to delete a task");
+        System.out.println("    What can I do for you?");
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     *Creates a new ToDo object of Task class
+     *adds ToDo Object to list of tasks
+     *prints a message to tell the user ToDo object has been added to TaskList
+     *
+     * @param line the input entered by user
+     * @param taskLetter the letter denoting the type of object
+     * @param list list of tasks
+     * */
+    public static void createToDoTask(String line , char taskLetter , TaskList list){
+        if (line.substring(TODO_INDEX).trim().isEmpty()){
+            System.out.println("    Todo task cannot be whitespaces or empty");
+            printCommandGuide();
+        } else {
+            Task task = new ToDo(line.substring(TODO_INDEX));
+            //taskLetter = Character.toUpperCase(line.charAt(0));
```

Comment:
At this stage, you should not be pushing commented codes. If you don't need this line, you can consider removing it.